blinky/config/bbb.exs

```diff
@@ -1,12 +1,12 @@
 # configuration for Beaglebone Black (target bbb)
 import Config
 
-config :blinky, led_list: [:led0, :led1, :led2, :led3]
-
-config :nerves_leds,
-  names: [
-    led0: "beaglebone:green:usr0",
-    led1: "beaglebone:green:usr1",
-    led2: "beaglebone:green:usr2",
-    led3: "beaglebone:green:usr3"
-  ]
+config :blinky,
+  indicators: %{
+    default: %{
+      led0: "beaglebone:green:usr0",
+      led1: "beaglebone:green:usr1",
+      led2: "beaglebone:green:usr2",
+      led3: "beaglebone:green:usr3"
+    }
```

Comment:
The Beaglebone's configuration is a bit more verbose. The idea is that each indicator could include a red, green, and blue LED. If that were the case, Delux would be able to make colors. However, it's not, so there are just 4 separate green indicators like this:

```suggestion
    default: %{
      green: "beaglebone:green:usr0",
    },
    led1: %{
      green: "beaglebone:green:usr1",
    },
    led2: %{
      green: "beaglebone:green:usr2",
    },
    led3: %{
      green: "beaglebone:green:usr3"
    }
```